composer.lock

```diff
@@ -503,31 +503,31 @@
         },
         {
             "name": "nette/schema",
-            "version": "v1.3.0",
+            "version": "v1.2.5",
```

Comment:
I always need to run composer update because composer install does not work otherwise on my machine. 
Not sure if this is a good change?

Reply:
Will revert changes in this file